core/src/main/resources/schema/infinispan-config-11.0.xsd

```diff
@@ -641,21 +645,21 @@
     <xs:attribute name="gauges" type="xs:boolean" default="${GlobalMetrics.gauges}">
       <xs:annotation>
         <xs:documentation>
-          Enables exporting of gauge metrics. These are exported by default if statistics are enabled.
+          Exports gauge metrics. If you enable statistics, gauges are exported by default.
         </xs:documentation>
       </xs:annotation>
     </xs:attribute>
     <xs:attribute name="histograms" type="xs:boolean" default="${GlobalMetrics.histograms}">
       <xs:annotation>
         <xs:documentation>
-          Enables exporting of histograms. These are costly to compute so are not enabled by default.
+          Exports histograms. Disabled by default because histograms require additional computation.
```

Comment:
I would rephrase ```Exports histogram metrics. Histograms are not enabled by default because they require additional computation. Statistics must also be enabled or else no metrics will actually be exported. ```